UNRELEASED_CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Add `enforceUnique` parameter to `ChatClient::sendReaction` and `ChannelClient::sendReaction` methods .
 If reaction is sent with `enforceUnique` set to true, new reaction will replace all reactions the user has on this message.
 - Add suspending `setUserAndAwait` extension for `ChatClient`
+- Replace client event listener functions with ChatEventListener functional interface.
```

Comment:
Might want to expand this explanation with something about the old APIs being deprecated